exercises/takudzwa_makoni/ex01_basics/src/exercise01.cpp

```diff
@@ -0,0 +1,100 @@
+
+#include "pch.h"
+#include <iostream>
+#include <fstream>
+#include <list>
+#include <iomanip>
+#include <string>
+#include <algorithm>
+#include <cctype>
+#include <iomanip>
+
+
+using namespace std;
+list<string> searchList = {"which","holmes","there","could","photograph"}; // list of words to be searched.
+
+
+// return true if special character exists in c.
+bool isSpecialCharacter(char c)
+{
+	switch (c)
```

Comment:
This could be implemented using [`std::any_of`](https://en.cppreference.com/w/cpp/algorithm/all_any_none_of).

Where one exists it is advisable to make use of the STL algorithms if it cuts down on code you have to write or is more concise.

Reply:
have made these changes replacing switch/case 👍